src/Sulu/Bundle/AdminBundle/Resources/js/views/List/toolbarActions/ExportToolbarAction.js

```diff
@@ -125,6 +135,8 @@ export default class ExportToolbarAction extends AbstractListToolbarAction {
             enclosure: this.enclosure,
             newLine: this.newLine,
             ...this.listStore.options,
+            filter: filters,
+            search: search,
```

Comment:
Lint is failing this should just be:

```suggestion
            search,
```

Reply:
Thanks for the tip @alexander-schranz! Refactored in latest commit.

It seems Lint still failing, due to filter: filter?